manifests/pipeline.yaml

```diff
@@ -1,53 +1,67 @@
 apiVersion: v1
 kind: Template
 labels:
-  app: fedora-coreos
-  template: fedora-coreos-template
+  app: ${DISTRO}-coreos
+  template: ${DISTRO}-coreos-template
 metadata:
   annotations:
     description: |-
-      Jenkins pipeline for Fedora CoreOS.
+      Jenkins pipeline for CoreOS distributions.
     iconClass: icon-jenkins
     openshift.io/display-name: Fedora CoreOS Pipeline
     openshift.io/documentation-url: https://github.com/coreos/fedora-coreos-pipeline
     openshift.io/support-url: https://github.com/coreos/fedora-coreos-pipeline
-    openshift.io/provider-display-name: Fedora CoreOS
-    tags: fcos,jenkins,fedora
-  name: fedora-coreos
+    openshift.io/provider-display-name: fedora CoreOS
+    tags: ${TAGS}
```

Comment:
All this stuff BTW is just to make the template look pretty in the console template browser. We don't really have to support that though since one would normally set up the pipeline from the CLI, not the console. For `tags` here, I'd say it's not worth the overhead of a separate template parameter. Let's just extend it to e.g. `fcos,rhcos,jenkins,fedora`?

Reply:
Done